_rules/iframe-accessible-name-cae760.md

```diff
@@ -123,15 +131,23 @@ This page has no `iframe` element.
 
 #### Inapplicable Example 2
 
-This `iframe` is neither part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.
+This `iframe` is not [included in the accessibility tree][] because of setting a style of `display: none;`.
 
 ```html
-<iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
+<iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
 ```
 
 #### Inapplicable Example 3
 
-This `iframe` is not part of [sequential focus navigation][] because it has `tabindex="-1"` and not [included in the accessibility tree][] because of `role="presentation"`
+This `iframe` is not [included in the accessibility tree][] because of `role="presentation"`.
+
+```html
+<iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
+```
```

Comment:
The code of this example is the same as the previous (`display:none`) and doesn't match the description.

The description is the same as Failed Example 4 (just swapping `role="none"` for the equivalent `role="presentation"`) making it super weird that one fails and one is inapplicable.